contracts/Swap/SwapRouter.sol

```diff
@@ -331,11 +331,14 @@ contract SwapRouter is Ownable2StepUpgradeable, ReentrancyGuardUpgradeable, Rout
         address[] calldata path,
         uint256 deadline
     ) external payable override ensure(deadline) {
-        uint256 balanceBefore = IERC20(path[path.length - 1]).balanceOf(address(this));
+        uint256 balanceBefore = address(this).balance;
         _swapExactTokensForETH(amountIn, amountOutMin, path, address(this), TypesOfTokens.NON_SUPPORTING_FEE);
-        uint256 balanceAfter = IERC20(path[path.length - 1]).balanceOf(address(this));
+        uint256 balanceAfter = address(this).balance;
         uint256 swapAmount = balanceAfter - balanceBefore;
-        _repay(path[path.length - 1], vTokenAddress, swapAmount);
+        uint256 response = IVToken(vTokenAddress).repayBorrowBehalf(msg.sender, swapAmount);
```

Comment:
For vBNB the method `repayBorrowBehalf` doesn't accept the amount as a parameter. It has to be sent as `msg.value`. See it [here](https://github.com/VenusProtocol/venus-protocol/blob/develop/contracts/Tokens/VTokens/VBNB.sol#L112)

Reply:
Oops, my mistake. FIxed, can verify at [commit](https://github.com/VenusProtocol/venus-protocol/pull/215/commits/7e0d572778b51fbd246174bd16cbf5d1788306f8).